python/pyarrow/tests/test_exec_plan.py

```diff
@@ -28,6 +28,24 @@
 pytestmark = pytest.mark.dataset
 
 
+class UuidType(pa.PyExtensionType):
```

Comment:
Given that this is also in `test_extension_type.py` do we want to make a common place for this and import instead of repeating?  I don't feel too strongly here as it isn't much repetition.

Reply:
Oh yeah good call. I was actually not using `UuidType`. Switched to importing the type from `test_extension_type.py`.